Package.resolved

```diff
@@ -1,61 +0,0 @@
-{
```

Comment:
Please revert the removal of this file. We'd like to keep in the repository to make sure that our dependency tree is consistent across checkouts.

Reply:
Yep. My mistake.